docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -114,4 +114,14 @@ By using separate templates for each flow, a more clear distinction exist as to
 and potentially introduce unexpected changes and behavior that should only affect pages for a specific flow.
 
 If you have customizations to the `login-update-profile.ftl` template to customize how users update their profiles when authenticating through a broker, make sure to move your changes
-to the new template.
\ No newline at end of file
+to the new template.
+
+= Changes in jboss-logging event messages
```

Comment:
If we are going to backport this (which I assume that yest), then it should rather use file `changes-23_0_5.adoc` to add this release note?